src/modules/returnToNCI/returnToNCI-bar.js

```diff
@@ -255,15 +257,24 @@ var merge = function() {
             // var barStyles = '<link rel="stylesheet" href="/modules/returnToNCI/returnToNCI-bar.css" />';
 
 
+            var target = "_parent";
+
+            if (settings.uxWindow !== false) {
+                target = "_blank";
+            }
+            if (settings.sameUxWindow !== false) {
```

Comment:
Remove all references to settings.sameUxWindow.

Reply:
I think what Bryan is getting at here is `setting.uxWindow` should check for truthiness. If it's not `false` then check if the value is exactly `true` and if not then assign a potential string value to target. Otherwise the target will be `_black`. Like so: `target = target !== true ? setting.uxWindow : '_blank'`

Reply:
Uh, yeah, more so, 
target = setting.uxWindow === false ? '_blank' : setting.uxWindow